<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Database</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>197</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="okPushButton">
     <property name="minimumSize">
      <size>
       <width>88</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>88</width>
       <height>34</height>
      </size>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="cancelPushButton">
     <property name="minimumSize">
      <size>
       <width>88</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>88</width>
       <height>34</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="TabDbSelectorWidget" name="tabDbSelectorWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DatabaseParameterWidget" name="databaseParameterWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DatabaseParameterWidget</class>
   <extends>QWidget</extends>
   <header>DsgTools.gui.CustomWidgets.DatabasePropertiesWidgets.BasicPropertyWidgets.databaseParameterWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabDbSelectorWidget</class>
   <extends>QWidget</extends>
   <header>DsgTools.gui.CustomWidgets.SelectionWidgets.tabDbSelectorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
